src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -2563,10 +2575,10 @@ class CgroupUtils:
                 if 'memory' in jra:
                     if 'limit_in_bytes' in jra['memory']:
                         memory = size_as_int(jra['memory']['limit_in_bytes'])
-                pbs.logmsg(pbs.EVENT_DEBUG3, "cpus: %s" % cpus)
-                pbs.logmsg(pbs.EVENT_DEBUG3, "sockets: %s" % sockets)
-                pbs.logmsg(pbs.EVENT_DEBUG3, "devices: %s" % devices)
-                pbs.logmsg(pbs.EVENT_DEBUG3, "memory limit: %sB" % memory)
+                pbs.logmsg(pbs.EVENT_DEBUG4, "cpus: %s" % cpus)
+                pbs.logmsg(pbs.EVENT_DEBUG4, "sockets: %s" % sockets)
+                pbs.logmsg(pbs.EVENT_DEBUG4, "devices: %s" % devices)
+                pbs.logmsg(pbs.EVENT_DEBUG4, "memory limit: %sB" % memory)
```

Comment:
Why print 4 lines to the log when one would suffice?

Reply:
We still want devices to be on its own line because it could be quite long.